parcllabs/common.py

```diff
@@ -1,6 +1,14 @@
 VALID_PROPERTY_TYPES = ["SINGLE_FAMILY", "CONDO", "TOWNHOUSE", "ALL_PROPERTIES"]
-VALID_PROPERTY_TYPES_FOR_UNIT_SEARCH = ["SINGLE_FAMILY", "CONDO", "TOWNHOUSE", "OTHER"]
-VALID_ENTITY_NAMES = ["AMH", "TRICON", "INVITATION_HOMES", "HOME_PARTNERS_OF_AMERICA"]
+VALID_PROPERTY_TYPES_UNIT_SEARCH = ["SINGLE_FAMILY", "CONDO", "TOWNHOUSE", "OTHER"]
+VALID_ENTITY_NAMES = [
```

Comment:
are these constants so that you can catch bad requests before they are sent to the API?

Reply:
Yes, they won't hit the api. they will throw an error through `Validators`.